content/millennium/r4/documents/document-reference.md

```diff
@@ -0,0 +1,111 @@
+* [Document status](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.docStatus){:target="_blank"}
+* [Document type](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.type){:target="_blank"}
+* [Document category](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.category){:target="_blank"}
+* [Subject (Patient)](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.subject){:target="_blank"}
+* [Author](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.author){:target="_blank"}
+* [Authenticator/verifying provider](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.authenticator){:target="_blank"}
+* [Document description/title]( http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.description){:target="_blank"}
+* [Attachment ContentType, Created date/time, Title, and URL (fully qualified link to the document)](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.content.attachment){:target="_blank"}
+* [Patient encounter]( http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.context.encounter){:target="_blank"}
+* [Document period]( http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.context.period){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:document_reference, :r4) %>
+
+## Search
+
+Search for DocumentReferences that meet supplied query parameters:
+
+    GET /DocumentReference?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name                     | Required?          | Type          | Description
+--------------------------|--------------------|---------------|----------------------------------------------------------------------------------------------------
+ `_id`                    | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
+ `patient`                | This, or `_id`     | [`reference`] | The specific patient to return DocumentReferences for. Example: `12345`
+ `encounter`              | N                  | [`reference`] | The Encounter in which the document was created. May be a list separated by commas. Example: `123,456`
+ `period`                 | N                  | [`date`]      | Time of service that is being documented. Must use the ge and lt prefixes. Example: `lt2017-01-5`
+ `type`                   | N                  | [`token`]     | The type of document. May be a list separated by commas. Example: `http://loinc.org|11488-4`.
+ `_count`                 | N                  | [`number`]    | The maximum number of results to include in a page. Example: `50`
+
+- The searching with the `period` parameter:
```

Comment:
add Implementation Notes: line

this bullet needs a re-write :)

Reply:
[Took care of both](https://github.com/cerner/fhir.cerner.com/pull/334/commits/5e38b77a2b8751100b9950d56d54324d1e102e2b#diff-6e001351a4f222ecfb0d8c7f4e55c2d5R58)

Reply:
I do like using asterisks, but I more meant that 'the searching with the period parameter' isn't an english sentence

Reply:
[Made it a sentence](https://github.com/cerner/fhir.cerner.com/pull/334/commits/aa34333db1d8bd104a3fa21a45fcc97abcf2dbd0#diff-6e001351a4f222ecfb0d8c7f4e55c2d5R60)